<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>189</width>
    <height>395</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>220</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>底座选择</string>
   </property>
   <widget class="QRadioButton" name="radioButton_Robotic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>107</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>栀音机械臂底座</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_5or4Axis">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>107</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>栀音V1底座</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>信息显示方式</string>
   </property>
   <widget class="QRadioButton" name="radioButton_QR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>89</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>二维码识别</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>89</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>OCR字符识别</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_None">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>89</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>无底版及信息</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>二维码参数</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>纠错率：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_correction">
       <item>
        <property name="text">
         <string>L-7%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>M-15%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Q-25%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H-30%</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>宽度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_qrwidth">
       <property name="minimum">
        <double>5.000000000000000</double>
       </property>
       <property name="maximum">
        <double>30.000000000000000</double>
       </property>
       <property name="value">
        <double>15.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
